R/derive_param_tte.R

```diff
@@ -505,6 +566,16 @@ derive_param_tte <- function(dataset = NULL,
 #'
 #'   Permitted Values:  `"first"`, `"last"`
 #'
+#'  @param check_type Check uniqueness
```

Comment:
```suggestion
#' @param check_type Check uniqueness
```
(If the tag is indented, it is rendered as text.)